data/json/items/gunmod/mechanism.json

```diff
@@ -85,5 +85,53 @@
     "install_time": "10 m",
     "min_skills": [ [ "weapon", 2 ] ],
     "flags": [ "WATERPROOF_GUN" ]
+  },
+  {
+    "id": "m249semi",
+    "type": "GUNMOD",
+    "name": { "str": "M249 semi-auto trigger" },
+    "description": "Trigger modification, that allow an m249 to work in semi-auto mode.",
+    "weight": "60 g",
+    "volume": "250 ml",
+    "integral_volume": "0 ml",
+    "price": 24000,
+    "price_postapoc": 350,
+    "material": [ "steel" ],
+    "symbol": ":",
+    "color": "red",
+    "location": "mechanism",
+    "mod_targets": [ "m249" ],
+    "install_time": "2 h",
+    "dispersion_modifier": 5,
+    "consume_chance": 50,
+    "consume_divisor": 110,
+    "//durability_modifier": -3,
+    "mode_modifier": [ [ "DEFAULT", "semi-auto", 1 ] ],
+    "min_skills": [ [ "mechanics", 4 ], [ "rifle", 5 ] ],
+    "flags": [ "INSTALL_DIFFICULT", "CONSUMABLE" ]
+  },
+  {
+    "id": "m240semi",
+    "type": "GUNMOD",
+    "name": { "str": "M240 semi-auto trigger" },
+    "description": "Trigger modification, that allow an m240 to work in semi-auto mode.",
```

Comment:
Again, maybe "A trigger modification that allows the M240 to fire in semi-automatic mode." would work better here.

Reply:
thanks, thats sound way better